ouroboros-consensus/test/infra-test/Test/Ouroboros/Consensus/PeerSimulator/Trace.hs

```diff
@@ -0,0 +1,68 @@
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE NamedFieldPuns #-}
+
+module Test.Ouroboros.Consensus.PeerSimulator.Trace (
```

Comment:
Should we have a module of Tracer. It looks a bit like having a module of `Types`, which has been argued to be an anti-pattern.

Reply:
I found it noisy in the other module…FWIW, there are two other instances of tracer modules in the project 🙂

Reply:
Ideally the pattern would be to have tracing stuff near what they are tracing for. However, those tracing utilities are “orphan”. I'd rather keep them in a separate module as they are now or move them to an `Extras` module in the right place, eg. `*.ChainDb.Extras`.